vocode/streaming/synthesizer/cartesia_synthesizer.py

```diff
@@ -83,17 +119,32 @@ async def create_speech_uncached(
         is_first_text_chunk: bool = False,
         is_sole_text_chunk: bool = False,
     ) -> SynthesisResult:
-        generator = await self.client.tts.sse(
-            model_id=self.model_id,
-            transcript=message.text,
-            voice_id=self.voice_id,
-            stream=True,
-            output_format=self.output_format,
-        )
+        await self.initialize_ws()
+
+        if is_first_text_chunk:
+            await self.ctx.no_more_inputs()
```

Comment:
awesome! this makes a lot of sense - so should the websocket stay alive for the duration of the phone call? or should one be opened per turn?

Reply:
The websocket should stay alive for the duration of the phone call! We only need a new continuation `context` per turn. 

We can refresh the websocket connection in case it dies.

Reply:
got it, does the websocket refresh happen within the SDK?

Reply:
yes! any `send` will refresh the websocket